.github/workflows/ci.yml

```diff
@@ -6,10 +6,17 @@ jobs:
   test:
     name: Julia ${{ matrix.version }} - ${{ matrix.os }} - ${{ matrix.arch }} - ${{ github.event_name }}
     runs-on: ${{ matrix.os }}
+    continue-on-error: ${{ matrix.version == 'nightly' }}
     strategy:
       matrix:
-        version: ['1.6', '^1.7.0-rc1', 'nightly']
-        os: [ubuntu-latest, windows-latest, macOS-latest]
+        version: 
+          - '1.6'
+          - '^1.7.0-0'
+          - 'nightly'
+        os: 
+          - ubuntu-latest
+          - windows-latest
+          - macOS-latest
```

Comment:
It looks like macOS VM is problematic. Should we keep it?

Reply:
Keep it, why not? I suspect it's the cache problem and we can fix that later.